sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TablesExtensionConfigProvider.cs

```diff
@@ -120,6 +125,25 @@ private async Task<TableClient> CreateTableClient(TableAttribute attribute, Canc
             return table;
         }
 
+        internal Task<ParameterBindingData> CreateParameterBindingData(TableAttribute attribute, ValueBindingContext context)
+        {
+            var tableDetails = new TablesParameterBindingDataContent(attribute);
+            var tableDetailsBinaryData = new BinaryData(tableDetails);
+            var parameterBindingData = new ParameterBindingData("1.0", Constants.ExtensionName, tableDetailsBinaryData, "application/json");
+
+            return Task.FromResult(parameterBindingData);
+        }
+
+        internal Task<ParameterBindingData> CreateCustomParameterBindingData(TableAttribute attribute, string rowKey)
+        {
+            var tableDetails = new TablesParameterBindingDataContent(attribute);
```

Comment:
nit - Not sure if Table**s** in `TablesParameterBindingDataContent` is intended or should it be TableParameterBindingDataContent?

Reply:
Since the extension is called `Microsoft.Azure.WebJobs.Extensions.Tables` I decided to keep it consistent with that